.github/workflows/run-tests.yml

```
@@ -41,3 +41,27 @@ jobs:
         if: always()
         run: |
           docker stop pyk-ci
+  run-tests-nix:
+    name: 'Run Tests Nix'
```

What do you think, would this be a better name for the job?

```suggestion
  build-on-nix:
    name: 'Build on Nix'
```